weaver/processes/wps_package.py

```diff
@@ -708,6 +718,37 @@ def _update_package_metadata(wps_package_metadata, cwl_package_package):
             set(wps_package_metadata.get("keywords", [])) | set(cwl_package_package.get("s:keywords", []))
         )
 
+    # specific use case with a different mapping
+    if "s:version" in cwl_package_package:
+        wps_package_metadata["version"] = str(
+             str(cwl_package_package.get("s:version", "")) or str(wps_package_metadata.get("version"))
```

Comment:
I think those must be switched around. The process `version` explicitly provided takes priority over the CWL one.